README.md

```diff
@@ -11,13 +11,13 @@ Tekton!
 
 Download the latest binary executable for your operating system:
 
-* [Mac OS X](https://github.com/tektoncd/cli/releases/download/v0.2.2/tkn_0.2.2_Darwin_x86_64.tar.gz)
```

Comment:
wonder if this can be replaced with `latest` instead of a version number?

Reply:
:D https://github.com/tektoncd/cli/pull/267#issuecomment-527424693